src/main/groovy/org/shipkit/internal/notes/header/HeaderProvider.java

```diff
@@ -0,0 +1,11 @@
+package org.shipkit.internal.notes.header;
+
+public class HeaderProvider {
+
+    public static final String HEADER_START = "<sup><sup>";
```

Comment:
It's pretty smart to use the tags! They are not visible I assume in the text on GitHub?